apps/web/src/ee/clerk/components/OrganizationSwitcher.tsx

```diff
@@ -0,0 +1,21 @@
+import { OrganizationSwitcher as ClerkOrganizationSwitcher } from '@clerk/clerk-react';
+
+// TODO: this is tmp. styling
+const OrganizationSwitcherAppearance = {
+  elements: {
+    organizationSwitcherTrigger: {
+      paddingInline: 'var(--nv-spacing-125)',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice! Can also use 'token(spacing.125)'‏